test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/ConsoleCodeGeneratorTests.cs

```diff
@@ -281,7 +291,7 @@ public void ModelInputClassTest()
                 }.TransformText(),
                 Name = "ModelInput.cs",
             };
-            NamerFactory.AdditionalInformation = "null_map";
+            NamerFactory.AdditionalInformation = info + "_null_map";
```

Comment:
I made this change to produce and check for 4 "approved" files within 1 unit test, instead of making a new unit test consisting of mostly duplicate code from this test.